app/src/main/java/org/mozilla/tv/firefox/telemetry/TelemetryIntegration.kt

```diff
@@ -294,6 +298,22 @@ open class TelemetryIntegration protected constructor(
         TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.FXA, Value.FXA_LOGIN_BUTTON).queue()
     }
 
+    fun fxaShowProfileButtonClickEvent() {
+        TelemetryEvent.create(Category.ACTION, Method.CLICK, Object.FXA, Value.FXA_SHOW_PROFILE_BUTTON).queue()
+    }
+
+    fun fxaProfileGetTabsButtonClickEvent() {
```

Comment:
It's not a "get tabs" button: it's a "show me the onboarding" button, right? Though the "Firefox tabs" naming is weird.

Reply:
I think it should match the text, but I'm also a little confused by the "Firefox Tabs" name.  @athomasmoz should we maybe change this string?  "Get Firefox tabs" maybe?